Foundation/String.swift

```diff
@@ -12,6 +12,10 @@
 
 import CoreFoundation
 
+extension String {
+  public static let transformStripCombiningMarks: CFString = kCFStringTransformStripCombiningMarks
+}
```

Comment:
Isn't this adding public API?

Reply:
hmm, yeah, I suppose it is.  I'll let @millenomi make the call on that.  I guess we can make this an `internal` extension and use `NS_FOUNDATION_TESTABLE_IMPORT` to block off the tests.

Reply:
All SPI should be internal and guarded by `@testable import` and friends.

Reply:
@millenomi - sounds good; wasn't sure if this was something that should be exposed or not

Reply:
In general, if it's not already documented on developer.apple.com, I do not have the power to approve patches that add new `public` stuff (modulo very specific circumstances.)

Reply:
Well, ignoring my failure at reading: this is documented: https://developer.apple.com/documentation/foundation/stringtransform/1413163-stripcombiningmarks

Reply:
#2090 addresses this, need to adjust the tests afterwards.